### DISCUSSION TITLE
iOS 端 Safari 浏览器看视频无法全屏

### DISCUSSION THREAD

**Discussion by wejudging** [spencerwooo/onedrive-vercel-index#579] [Q&A]
ios端safari浏览器看视频无法全屏

**Comment by spencerwooo** [comment]
视频播放器用的是 `plyr`，其中全屏相关的描述 - https://github.com/sampotts/plyr#fullscreen

**Reply by wejudging** [reply]
我看了一下plyr官网的视频，也是同样的问题。ios的全屏浏览器上面地址栏不隐藏，导致无法全屏，可能是还没修复的bug吧。

**Reply by wejudging** [reply]
好像是需要改一下代码，全屏的时候调用safari本来的那种全屏，https://codepen.io/antonio_laguna/pen/GYzNmx

**Comment by wejudging** [comment]
https://github.com/sampotts/plyr/issues/1233

**Comment by aidenlx** [comment]
plyr的老问题了，可以在iOS端加个按钮开关`<video>`的playsinline属性允许恢复默认的播放时自动全屏

**Reply by aidenlx** [reply]
https://webkit.org/blog/6784/new-video-policies-for-ios/
